src/cloud_events.ts

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 import * as express from 'express';
-import {CloudEventsContext} from './functions';
+import {CloudEvent} from 'cloudevents';
```

Comment:
we should expose the CESDK interfaces in our APIs not the concrete class.

```
import {CloudEventV1 as CloudEvent} from 'cloudevents';
```

Reply:
Alternatively, as described below, pull the re-exported interface from `./functions`

```
import {CloudEvent} from './functions';
```